.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+(Thanks for sending a pull request. Please fill out the blanks below.)
+
+What does this implement/fix? Explain your changes
+---------------------------------------------------
+...
+
+Does this close any currently open issues\?
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-heading-punctuation] Don’t add a trailing `?` to headings](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4576380)